pkg/manager/manager.go

```diff
@@ -92,6 +92,7 @@ func (k *KubeMacPoolManager) Run(rangeStart, rangeEnd net.HardwareAddr) error {
 			return errors.Wrap(err, "unable to set up manager")
 		}
 
+		os.Setenv("KUBEVIRT_CLIENT_GO_SCHEME_REGISTRATION_VERSION", "v1")
```

Comment:
you'll need to set this environment variable before executing the binary. So set it on the pod's container spec, or using an entry point shell script. 

setting it within the binary is too late as the init function that interprets the env var will have already executed.

Reply:
I see, thanks!